frontend/components/Export.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+import { IframeStamper } from "@turnkey/iframe-stamper";
+import { Dispatch, SetStateAction, useEffect, useState } from "react";
+
+interface ExportProps {
+  iframeUrl: string;
+  iframeDisplay: string;
+  setIframeStamper: Dispatch<SetStateAction<IframeStamper | null>>;
+}
+
+const TurnkeyIframeContainerId = "turnkey-iframe-container-id";
+const TurnkeyIframeElementId = "turnkey-iframe-element-id";
```

Comment:
Is this going to clash with the recovery iframe IDs? I believe they are the same.

Reply:
good catch, updating!